app/index.js

```diff
@@ -7,7 +7,7 @@
 
 var path = require('path');
 var SG = require('strong-globalize');
-SG.SetRootDir(path.join(__dirname, '..'));
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```

Comment:
>Is it an entry point to a loopback app?

No I don't think if `loopback-datasource-juggler` is the entry point.

I was just thinking of `loopback-datasource-juggler` based on this comment:
>We don't need to update all repos. Just the top modules. I will probably add it to loopback as well.

So if it is just for entry point modules, then I don't think if we needed it for `loopback-datasource-juggler` and we are good.

Thanks!